packages/space-opera/src/test/hotspot_panel/hotspot_editor_test.ts

```diff
@@ -54,10 +54,8 @@ describe('hotspot editor test', () => {
     expect(hotspots[0].annotation).toBe('new annotation');
   });
 
-  it('fires dispatchRemoveHotspot when remove button is clicked', () => {
-    const removeButton = hotspotEditor.shadowRoot!.querySelector(
```

Comment:
this button was moved further down, so just testing the function is just as good, as opposed to testing the button itself